rules-reviewed/jboss-eap/websphere/tests/websphere-mq.windup.test.xml

```diff
@@ -0,0 +1,500 @@
+    <testDataPath>samples/WebSphereMQ</testDataPath>
+    <rulePath>../websphere-mq.windup.xml</rulePath>
+    <ruleset id="websphere-mq" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+        <metadata>
+            <description>
+                Provides analysis of WebSphere JMS proprietary classes and constructs that may require individual attention when migrating to JBoss EAP 6+.
+            </description>
+            <dependencies>
+                <addon id="org.jboss.windup.rules,windup-rules-javaee,2.2.0.Final"/>
+                <addon id="org.jboss.windup.rules,windup-rules-java,2.2.0.Final"/>
+            </dependencies>
+            <sourceTechnology id="websphere" versionRange="(0,)"/>
+            <sourceTechnology id="websphere-mq" versionRange="(0,)"/>
+            <targetTechnology id="eap" versionRange="[6,)"/>
+            <targetTechnology id="jms" versionRange="[6,)"/>
+            <tag>reviewed-2015-06-05</tag>
+            <tag>messaging</tag>
+            <tag>mq</tag>
+            <tag>mqe</tag>
+            <tag>ibm</tag>
+            <tag>jms</tag>
+        </metadata>
+
+        <rules>
+            <!-- WebSphere JMS MQe -->
+
+            <!-- Package-matching generic rule - MQ. -->
+            <rule id="websphere-mq-02500-test-c1">
+                <when>
+                    <not>
+                        <classification-exists classification="WebSphere MQ API"/>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="Classification not found: WebSphere MQ API"/>
+                </perform>
+            </rule>
+
+            <!-- Example code to match:
+                Properties env = new Properties();
+                env.put(
+                    Context.INITIAL_CONTEXT_FACTORY,
+                    "com.ibm.mq.jms.context.WMQInitialContextFactory");
+            -->
+            <rule id="websphere-mq-03000-test-c1">
+                <when>
+                    <not>
+                        <classification-exists classification="WebSphere MQ JMS InitialContextFactory"/>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="Classification not found: WebSphere MQ JMS InitialContextFactory"/>
+                </perform>
+            </rule>
+            <rule id="websphere-mq-03000-test-h1">
+                <when>
+                    <not>
+                        <hint-exists message="For EAP 6, migrate to JBoss HornetQ."/>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="Hint not found: For EAP 6, migrate to JBoss HornetQ."/>
+                </perform>
+            </rule>
+
+            <!-- .scp config files - WINDUPRULE-1 -->
```

Comment:
Is this comment necessary when the rule-id is now specified in the test rule-id? Please remove.